pennylane/qchem/openfermion_obs.py

```diff
@@ -802,6 +1001,7 @@ def decompose(hf_file, mapping="jordan_wigner", core=None, active=None):
 def molecular_hamiltonian(*args, **kwargs):
     """molecular_hamiltonian(molecule, method="dhf", active_electrons=None, active_orbitals=None,\
     mapping="jordan_wigner", outpath=".", wires=None, args=None, convert_tol=1e12)
+
```

Comment:
Why this is changes? The PR is not synced with master?

Reply:
This seems to be a sync issue as I started from qchem_feature branch, I have changed this.